tes-adapter/build.gradle

```diff
@@ -1,16 +1,37 @@
-plugins {
-    id 'java'
-}
+group 'com.epam.pipeline.adapter'
```

Comment:
group should be  group 'com.epam.pipeline'

Reply:
fixed